frontend/src/store/modules/product.js

```diff
@@ -1,27 +1,53 @@
-const product = {
+import axios from 'axios';
 
+const product = {
     namespaced: true,
-
     state: { 
-        newProduct: undefined
+        newProduct: undefined,
```

Comment:
Instead of making this `undefined` let's just make it `null`. I know it still _technically_ works as `undefined` but `null` is a better indicator that the state is being initialised, but with no value